src/chains/models.py

```diff
@@ -147,16 +149,37 @@ class GasPrice(models.Model):
     rank = models.SmallIntegerField(
         default=100
     )  # A lower number will indicate higher ranking
+    max_fee_per_gas = Uint256Field(
+        verbose_name="Max fee per gas (wei)", blank=True, null=True
+    )  # type: ignore[no-untyped-call]
+    max_priority_fee_per_gas = Uint256Field(
+        verbose_name="Max priority fee per gas (wei)", blank=True, null=True
+    )  # type: ignore[no-untyped-call]
 
     def __str__(self) -> str:
-        return f"Chain = {self.chain.id} | uri={self.oracle_uri} | fixed_wei_value={self.fixed_wei_value}"
+        return f"Chain = {self.chain.id} | uri={self.oracle_uri} | fixed_wei_value={self.fixed_wei_value} | max_fee_per_gas={self.max_fee_per_gas} | max_priority_fee_per-gas={self.max_priority_fee_per_gas}"  # noqa E501
 
     def clean(self) -> None:
-        if (self.fixed_wei_value is not None) == (self.oracle_uri is not None):
+        fixed_wei_defined = self.fixed_wei_value is not None
+        fixed1559_defined = (
+            self.max_fee_per_gas is not None
+            and self.max_priority_fee_per_gas is not None
+        )
+        oracle_defined = self.oracle_uri is not None
+        exactly_one_variant = reduce(
+            xor, [fixed_wei_defined, fixed1559_defined, oracle_defined]
+        )
```

Comment:
This won't work as it is since `True XOR True XOR True` becomes `False XOR True` which becomes `True` and that won't match with "exactly one variant".

Reply:
Maybe something like:

```python3
exactly_one_variant = [fixed_wei_defined, fixed1559_defined, oracle_defined].count(True) == 1
```

wdyt?

Reply:
Whoops I missed that one. Your suggestions sounds good. I will change it to that :)